src/Raven.Studio/typescript/common/shell/continueTest.ts

```diff
@@ -28,7 +28,7 @@ class continueTest {
     continue() {
         const doc = document.empty();
         const db = databasesManager.default.getDatabaseByName(this.databaseName());
-        new saveDocumentCommand("Debug/Done", doc, db, false)
+        new saveDocumentCommand("Debug/Done", doc, db, false, false)
```

Comment:
no need to add this extra param - use default

Reply:
done